src/UglyToad.PdfPig/Content/Letter.cs

```diff
@@ -160,6 +172,35 @@ private TextOrientation GetTextOrientation()
             return TextOrientation.Other;
         }
 
+        private TextOrientation GetTextOrientationRot()
+        {
+            double rotation = GlyphRectangle.Rotation;
+            int rotationInt = (int)Math.Round(rotation, MidpointRounding.AwayFromZero);
+
+            if (Math.Abs(rotation - rotationInt) >= 10e-5)
+            {
+                return TextOrientation.Other;
+            }
+
+            switch (rotationInt)
+            {
+                case 0:
+                    return TextOrientation.Horizontal;
+
+                case -90:
+                    return TextOrientation.Rotate90;
+
+                case 180:
+                case -180:
+                    return TextOrientation.Rotate180;
+
+                case 90:
+                    return TextOrientation.Rotate270;
+            }
+
+            throw new Exception($"Could not find TextOrientation for rotation '{rotation}'.");
```

Comment:
Not sure I follow this change but is there a risk that e.g. 270 throws, should we return `Other` here instead of throwing? Or 270 is just handled by the 90/-90 cases? Should we check that the value is divisible by 90 cleanly?

Reply:
Hi @EliotJones, apologies I completely missed your comment here...

You are 100%, this needs to be handled differently. I will push a fix shorlty